components/server/src/workspace/workspace-starter.ts

```diff
@@ -1552,7 +1554,21 @@ export class WorkspaceStarter {
             spec.setTimeout(defaultTimeout);
             spec.setMaximumLifetime(workspaceLifetime);
             if (allowSetTimeout) {
-                if (user.additionalData?.workspaceTimeout) {
+                const organizationSettings = await this.orgService.getSettings(user.id, workspace.organizationId);
+                if (organizationSettings.timeoutSettings?.inactivity) {
+                    try {
+                        const timeout = WorkspaceTimeoutDuration.validate(
+                            organizationSettings.timeoutSettings.inactivity,
+                        );
+                        spec.setTimeout(timeout);
+                    } catch (err) {}
+                }
+
+                // Users can optionally override the organization-wide timeout default if the organization allows it
+                if (
+                    organizationSettings.timeoutSettings?.allowChangeByMembers !== false &&
```

Comment:
This should be `=== true`, and we should handled the null/default mapping in the service.

Reply:
Here we need to check allowChangeByMembers as well: https://github.com/gitpod-io/gitpod/blob/b43c97ed9a7e81a0568e237ef0d267fb312a317a/components/server/src/workspace/workspace-service.ts#L915